templates/docker-compose.yml.j2

```diff
@@ -48,6 +48,7 @@ services:
       --max-connections={{ nim_waku_p2p_max_connections }}
       --store-capacity={{ nim_waku_store_capacity }}
       --dns4-domain-name={{ nim_waku_dns4_domain_name }}
+      --rln-relay-content-topic={{nim_waku_rln_relay_content_topic}}
```

Comment:
Shouldn't this flag be conditional depending on whether `rln-relay` protocol is enabled?

Or is there no issue if you supply flags for disabled protocols? It's just messy?

Reply:
@jakubgs can you please let me know if the added `if` statement looks correct to you? thanks

Reply:
Yes, this format of `if` condition should work just fine, BUT you're not closing it. You also need an `{% endif %}`.

Also, I'd use double quotes in Jinja2 code blocks, like in the rest of the file.

Reply:
Thanks, addressed in https://github.com/status-im/infra-role-nim-waku/pull/1/commits/87e2849a4b0e08e2b1a482ae0b9ba0a5e34e7798

Reply:
I just realized some of the variables are enclosed in single quotes, it this also necessary for `nim_waku_rln_relay_content_topic`? this key holds a string value as well @jakubgs

Reply:
Depends. In most cases string values in YAML can be passed without quoting, BUT if you're doing interpolation with `{{ }}` then it has to be quotes. There are also some other edge cases with special characters where strings have to be quoted. As a general rule I try to quote everywhere, but sometimes I miss things.

Reply:
Also, please add spaces around `nim_waku_rln_relay_content_topic` here.

Reply:
Sure, done in https://github.com/status-im/infra-role-nim-waku/pull/1/commits/4c71b953d326f1f12982174eb42020faf07b865f